src/openms/source/SYSTEM/File.cpp

```diff
@@ -621,8 +621,12 @@ namespace OpenMS
   Param File::getSystemParameters()
   {
     String home_path = File::getOpenMSHomePath();
-
+    //Comply with https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html on unix identifying systems
+    #ifdef __unix__
+    String filename = home_path + "/.config/OpenMS/OpenMS.ini";
```

Comment:
technically, you'd need to query `$XDG_CONFIG_HOME` first and use that as a full prefix. 
I'm not sure though if its usually defined or if the fallback is used in most cases anyway....

From the docs:
```
$XDG_CONFIG_HOME defines the base directory relative to which user-specific configuration files should be stored. If $XDG_CONFIG_HOME is either not set or empty, a default equal to $HOME/.config should be used. 
```